requirements.txt

```diff
@@ -1,2 +1,2 @@
-setuptools==26.1.1
+setuptools>=36.2
```

Comment:
Running buildout installs setuptools==39.1.0, see [this](https://travis-ci.org/ironmaniiith/plone.importexport/builds/542305258#L501)